subgraphs/aave-v2-forks/protocols/radiant/src/constants.ts

```diff
@@ -0,0 +1,28 @@
+import { Network } from "../../../src/constants";
+
+/////////////////////
+///// Addresses /////
+/////////////////////
+
+export const REWARD_TOKEN_ADDRESS =
+  "0x0c4681e6c0235179ec3d4f4fc4df3d14fdd96017"; // RDNT token
+export const RDNT_WETH_Uniswap_Pair =
+  "0x24704aff49645d32655a76df6d407e02d146dafc"; // RDNT/WETH
+export const RWETH_ADDRESS = "0x15b53d277af860f51c3e6843f8075007026bbb3a";
+
+/////////////////////////////
+///// Protocol Specific /////
+/////////////////////////////
+
+export namespace Protocol {
+  export const NAME = "Radiant Captital";
+  export const SLUG = "radiant-capital";
```

Comment:
we should have the slug consistent across the folder name and the deployment, so should be `radiant` instead